qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -99,43 +98,81 @@ def _default_experiment_options(cls) -> Options:
                 by doing :code:`options.gate_class()`.
             normalization (bool): If set to True the DataProcessor will normalized the
                 measured signal to the interval [0, 1]. Defaults to True.
-            add_sx (bool): If True then the circuits will start with an sx gate. This is typically
-                needed when calibrating pulses with a target rotation angle of :math:`\pi`. The
-                default value is False.
-            add_xp_circuit (bool): If set to True then a circuit with only an X gate will also be
-                run. This allows the analysis class to determine the correct sign for the amplitude.
+            add_cal_circuits (bool): If set to True then two circuits to calibrate 0 and 1 points
+                will be added. These circuits are often needed to properly calibrate the amplitude
+                of the ping-pong oscillation that encodes the errors. This helps account for
+                state preparation and measurement errors.
         """
         options = super()._default_experiment_options()
-        options.repetitions = list(range(15))
+        options.repetitions = list(range(1, 15))
         options.gate = None
         options.normalization = True
-        options.add_sx = False
-        options.add_xp_circuit = True
+        options.add_cal_circuits = True
 
         return options
 
-    def __init__(self, qubit: int, gate: Gate, backend: Optional[Backend] = None):
+    def __init__(self, qubits: Sequence[int], gate: Gate, backend: Optional[Backend] = None):
         """Setup a fine amplitude experiment on the given qubit.
 
         Args:
-            qubit: The qubit on which to run the fine amplitude calibration experiment.
+            qubits: The qubit(s) on which to run the fine amplitude calibration experiment.
             gate: The gate that will be repeated.
             backend: Optional, the backend to run the experiment on.
         """
-        super().__init__([qubit], analysis=FineAmplitudeAnalysis(), backend=backend)
+        super().__init__(qubits, analysis=FineAmplitudeAnalysis(), backend=backend)
         self.set_experiment_options(gate=gate)
 
-    def _pre_circuit(self) -> QuantumCircuit:
+    def _spam_cal_circuits(self, meas_circuit: QuantumCircuit) -> List[QuantumCircuit]:
+        """This method returns the calibration circuits.
+
+        Calibration circuits allow the experiment to overcome state preparation and
+        measurement errors which cause ideal probabilities to be below 1.
+
+        Args:
+            meas_circuit: The measurement circuit, so that we only apply x gates to the
+                measured qubits.
+        """
+        cal_circuits = []
+
+        for add_x in [0, 1]:
+            circ = QuantumCircuit(self.num_qubits, meas_circuit.num_clbits)
+
+            if add_x:
+                qubits = meas_circuit.get_instructions("measure")[0][1]
```

Comment:
probably we can explicitly add this as constructor argument of the baseclass?

```python
..., meas_qubit: Optional[int] = None
```

Then single qubit fine amp can set the same index with qubit, while two qubit fine amp can set second qubit here. I think tomography does the same thing and looks good to me.
https://github.com/Qiskit/qiskit-experiments/blob/d6957f4a63799da160f5f2a581d192e5db271f74/qiskit_experiments/library/tomography/tomography_experiment.py#L58

Reply:
Good idea. See here: [ce88a45](https://github.com/Qiskit/qiskit-experiments/pull/655/commits/ce88a459db6cd0187d018d2536bb16f484c4b3fd)